### DISCUSSION TITLE
More precise authorisations for teams and users

### DISCUSSION THREAD

**Discussion by KarlKeu00** [pretix/pretix#4224] [Ideas] [answered]
Are there plans to authorise teams more specifically on certain topics? More specifically, I was thinking of the following examples or restrictions:
- Viewing/evaluating answers to questions without being able to edit products
- Making internal comments on orders without being able to edit orders
- Entering payments without being able to edit orders
- Restrict editing of your own (admin) account (especially relevant for e.g. SSO login)
- Authorisation to send emails

**Comment by raphaelm** [comment, marked as answer]
Yes, see #3461 for our plans
